charon/src/bin/charon-driver/translate/translate_types.rs

```diff
@@ -1,4 +1,6 @@
 use super::translate_ctx::*;
+use charon_lib::ast::Literal::*;
+use charon_lib::ast::types::LiteralTy::*;
```

Comment:
Same here, avoid glob imports of variants unless they're used a lot in the file.